### DISCUSSION TITLE
PRO vs FOSS

### DISCUSSION THREAD

**Discussion by Kay-Kostrowski** [d4rken-org/sdmaid-se#284] [Q&A] [answered]
I just bought the new version SE for my collection of fine apps, some days ago.
It showed the blue "pro" inside the app.
Today there was an advice inside the app, that a new beta update is available, so I followed and installed it.
The blue "pro" is gone and it shows some yellow "FOSS" now.

What is the difference between both PRO and FOSS?

Thanks for keeping me clean!

**Comment by d4rken** [comment, marked as answer]
There is no functional difference. The Google Play version shows "PRO", while the GitHub version shows "FOSS".


What update did you previously have? The Google Play version doesn't have an "Update info". :thinking: 

Did you have you uninstall the old version or could you just install the APK fro GitHub?

**Reply by Kay-Kostrowski** [reply]
Thanks for your quick reply!


Yes, there was a message about not able to update, because of some interfering other version.

I got 0.7.09beta1 before and now 0.7.10beta0

I thought it might be a little bumpy because of early release and beta version etc. so I just deinstalled 09beta1 and installed 10beta0.
I am not sure if there was a realy an offical update hint,
or maybe just me looking around in that app. 😇
I found something interesting about change log in settings, I have highlighted in left pic. That linked to the list in right pic, with the higher version waving to me, "install me".

Is there anything you would benefiting from? Like debugging report or screenshots of overview from both SD Maid apps or some like that?
 I![AFFIX_20230515_163852.png](https://github.com/d4rken-org/sdmaid-se/assets/84748960/88b3a00d-d7b7-4c78-b827-423977247b52)

**Reply by d4rken** [reply]
It's important that there isn't a way for Google Play users to be told download an APK, that's against Play Store policies. So your initial description worried me, but I think you just meant the link to the change/release and not with the actual "update available" card in the app UI (which shouldn't exist in Google Play builds).

If you previously had the FOSS build from GitHub you should be able to just update over it, if you have to reinstall to update, after the next update is released, please raise an issue ticket.

**Reply by Kay-Kostrowski** [reply]
No "update card" at all and nothing told me to uodate anything. Just following that change/release link and got to that list!
Sorry for worrying you! I didn't know the importance, so I was unprecise, because spotting a new and higher version somewhere is like a hint for me to update... 😉

I dont't know how strict google is, but I easily switched from playstore app to that list in githup only by one click on chabge/release and without any notice.
Maybe you should establish a short popup warning.
You are now leaving the Google-sector. Do you want to continue? ...
So there won't be some google guy in future, clicking the same way as me and might think this is to close involved in your app or even part of it and not just an extern list on github.
